cls/SourceControl/Git/Utils.cls

```diff
@@ -1055,8 +1055,11 @@ ClassMethod ListItemsInFiles(ByRef itemList, ByRef err) As %Status
         set mappingCoverage = $order($$$SourceMapping(mappingFileType, ""))
 
         while (mappingCoverage '= ""){
-
-            set mappedRelativePath = $$$SourceMapping(mappingFileType, mappingCoverage)
+            if ($data($$$SourceMapping(mappingFileType, mappingCoverage)) = 1){
+                set mappedRelativePath = $$$SourceMapping(mappingFileType, mappingCoverage)
+            } else {
+                set mappedRelativePath = $$$SourceMapping(mappingFileType, mappingCoverage, "NoFolders")
```

Comment:
Has the value at the "NoFolders" subscript changed from a flag to a directory? (This would actually explain something I was confused about before.)

Reply:
It has. So that we can use a path with `NoFolders` mappings as well and discussed here: https://github.com/intersystems/git-source-control/issues/130#issuecomment-1043432587

Instead of having 2 entries, I just put both under one.

Reply:
Got it. My main concern is that this is backwards-incompatible; old configuration won't work properly. Why can't we have the flag and folder separate?

Reply:
To be clear: I think it's cleaner to keep the flag and folder separate.

Reply:
Actually thinking about this more I'm not sure what I think. Regardless I think we should automate the update from the old settings version to the new settings format on package update. (This will be done from an <Invoke> in module.xml.)

Reply:
At the time I thought there would be less confusion if they were in the same entry. I don't think that will be the case though. Should I change it to be in two separate entries? I can do that in a different PR and we can just close this one because we need to reevaluate if this issue will still be relevant at that point.

Reply:
To be backwards-compatible, the only thing we have to change in the settings is to add the extension as the path.

Reply:
Let's change it back to be two separate entries. Let's cancel this PR and reevaluate after splitting back out.